week04/code.js

```diff
@@ -0,0 +1,83 @@
+const btns = document.getElementById("btns");
+const start = document.getElementById("start");
+const reset = document.getElementById("reset");
+
```

Comment:
가위, 바위, 보 버튼도 getElementById로 가져옵니다.

```suggestion
const scissor = document.getElementById("scissor");
const rock = document.getElementById("rock");
const paper = document.getElementById("paper");
```